packages/opentelemetry-core/src/baggage/propagation/W3CBaggagePropagator.ts

```diff
@@ -58,7 +58,8 @@ export class W3CBaggagePropagator implements TextMapPropagator {
   }
 
   extract(context: Context, carrier: unknown, getter: TextMapGetter): Context {
-    const headerValue: string = getter.get(carrier, BAGGAGE_HEADER) as string;
+    const headerBaggage = getter.get(carrier, BAGGAGE_HEADER);
+    const headerValue: string = (headerBaggage instanceof Array)? headerBaggage[0]: headerBaggage as string;
```

Comment:
Why not Array.isArray?

Reply:
This simply grabs the first baggage. Why not concatenate the array together using a `','` and process that?

`['key1=val1;prop=val,key2=val2;prop;prop2', 'key3=val3']` can be concatenated to `'key1=val1;prop=val,key2=val2;prop;prop2,key3=val3'` without issue. The spec was actually designed specifically to allow this.

Reply:
@weyert agree-fixed
@dyladan changed to use join to be comply with the spec